ccdproc/tests/test_combiner.py

```diff
@@ -332,16 +332,52 @@ def test_combine_limitedmem_scale_fitsimages():
 
     np.testing.assert_array_almost_equal(avgccd.data, ccd_by_combiner.data, decimal = 4)
 
+
 #test the optional uncertainty function in average_combine
 def test_average_combine_uncertainty(ccd_data):
     ccd_list = [ccd_data, ccd_data, ccd_data]
     c = Combiner(ccd_list)
     ccd = c.average_combine(uncertainty_func=np.sum)
-    np.testing.assert_array_equal(ccd.uncertainty.array, np.sum(c.data_arr, 0))
+    uncert_ref = np.sum(c.data_arr, 0) / np.sqrt(3)
+    np.testing.assert_array_equal(ccd.uncertainty.array, uncert_ref)
 
 #test the optional uncertainty function in median_combine
 def test_median_combine_uncertainty(ccd_data):
     ccd_list = [ccd_data, ccd_data, ccd_data]
     c = Combiner(ccd_list)
     ccd = c.median_combine(uncertainty_func=np.sum)
-    np.testing.assert_array_equal(ccd.uncertainty.array, np.sum(c.data_arr, 0))
+    uncert_ref = np.sum(c.data_arr, 0) / np.sqrt(3)
+    np.testing.assert_array_equal(ccd.uncertainty.array, uncert_ref)
+
+
+# test resulting uncertainty is corrected for the number of images
+def test_combiner_uncertainty_average(ccd_data):
```

Comment:
Could drop the `ccd_data` argument since you don't use it. Same is true for the other new test below.